index.html

```diff
@@ -7,7 +7,98 @@
 </head>
 
 <body>
-    <h1 class="main__title">lesson_1</h1>
+    <header class="header">
+        <h1 class="header__target" style="visibility: hidden">Магазин мебели</h1>
+        <div class="container">
```

Comment:
```suggestion
        <div class="header__container">
```
обертки называют как правило либо container либо wrapper